css/index.css

```diff
@@ -1,3 +1,368 @@
+  margin-left: 0;
+}
+
+header nav a:last-child {
+  margin-right: 0;
+}
+
+header .logo {
+  display: inline-block;
+  margin-top: 25px;
+}
+
+/* cta */
+
+.cta .cta-text {
+  display: inline-block;
+  vertical-align: top;
+  font-size: 77px;
+  margin-top: 120px;
+  text-align: center;
+  padding: 0px 75px;
+}
+
+.cta .cta-text button {
+  background: white;
+  vertical-align: top;
+  margin-top: 6px;
+  padding: 8px 44px;
+  font-size: 17px;
+  border: 1px solid black;
+  cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+.cta img {
+  display: inline-block;
+  margin-top: 69px;
+}
+
+.main-content {
+  border-top: 1px solid black;
+  margin-top: 76px;
+  padding-top: 39px;
+}
+
+.main-content .top-content {
+  padding: 0 10px;
+}
+
+.main-content .top-content .text-content {
+  width: 410px;
+  display: inline-block;
+  margin-right: 34px;
+}
+
+.main-content .top-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.main-content .middle-img {
+  margin-top: 45px;
+}
+
+.main-content .bottom-content {
+  margin-top: 37px;
+  padding: 0 10px;
+}
+
+.main-content .bottom-content .text-content {
+  width: 275px;
+  display: inline-block;
+  margin-right: 12px;
+}
+
+.main-content .bottom-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.contact {
+  margin-top: 57px;
+  border-top: 1px solid black;
+  padding: 39px 10px 0;
+}
+
+.contact p {
+  margin: 20px 0;
+}
+
+footer {
+  margin: 32px 0;
+  text-align: center;
+  font-size: 14px;
+}
+
+/* CSS for tablet below */
+@media (max-width:800px) {
+  .container  {
+    width: 800px;
+    display: flex;
+    flex-direction: column;
+    align-items: center
+  }
+  
+  header  {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+  }
+
+  header nav  {
+    width: 800px;
+    display: flex;
+    justify-content: space-around;
+  }
+  
+  header nav a {
+    font-weight: 900;
+    margin-right: 20px;
+  }
+
+  header .logo  {
+    margin-top: 30px;
+    order: -1;
+  }
+
+  .cta img  {
+    display: none;
+  }
+
+  .cta  {
+    display: flex;
+    justify-content: center;
+  }
+
+  .cta .cta-text  {
+    margin-top: 75px;
+  }
+
+  .cta .cta-text button   {
+    font-size: 15px;
+  }
+
+  .main-content {
+    width: 760px;
+    margin-top: 29px;
+    display: flex;
+    flex-direction: column;
+  }
+
+  .main-content .top-content  {
+    display: flex;
+    line-height: 1.2;
+  }
+
+  .main-content .middle-img {
+    margin-top: 21px;
+    height: 158px;
+  }
+
+  .main-content .bottom-content {
+    display: flex;
+    line-height: 1.2;
+    margin-top: 30px;
+  }
+
+  .contact  {
+    margin-top: 47px;
+    width: 760px;
+  }
+
+  .contact p  {
+    margin: 13px 0px;
+  }
+
+  footer  {
+    margin: 42px 0px;
+  }
+}
+
+/* CSS for mobile below */
+@media (max-width:500px) {
+  .container  {
+    width: 500px;
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+  }
+
+  header  {
```

Comment:
Your 800px media query has:

>  header  {
    display: flex;
    align-items: center;
    flex-direction: column;
  }

The `header` element already has `display: flex` on it because of that media query. No need to add it in this one.